src/Components/Facility/models.tsx

```diff
@@ -229,3 +226,430 @@ export type ICD11DiagnosisModel = {
+    doctor_medical_council_registration: string;
+    skills: {
+      id: string;
+      name: string;
+      description: string;
+    }[];
+  }[];
+};
+
+export type ILocalBodyResponse = StateModel & {
+  body_type: number;
+  localbody_code: string;
+  district: number;
+};
+
+export interface InventoryItemObjectModel {
+  id: number;
+  default_unit: UnitModel;
+  allowed_units: UnitModel[];
+  tags: UnitModel[];
+  name: string;
+  description: string;
+  min_quantity: number;
+}
+
+export type IInventorySummaryResponse = PaginatedResponse<{
+  id: string;
+  item_object: InventoryItemObjectModel;
+  unit_object: UnitModel;
+  created_date: string;
+  quantity: number;
+  is_low: boolean;
+  item: number;
+}>;
+
+export type IInventoryLogResponse = Omit<
+  IInventorySummaryResponse,
+  "results"
+> & {
+  results: {
+    id: string;
+    item_object: InventoryItemObjectModel;
+    unit_object: {
+      id: number;
+      name: string;
+    };
+    external_id: string;
+    current_stock: number;
+    quantity_in_default_unit: number;
+    is_incoming: boolean;
+    probable_accident: boolean;
+    created_by: number;
+  }[];
+};
+
+export type IFaciclityMinimumQuantityRequest = {
+  min_quantity?: number;
+  item?: number;
+};
+export type IFaciclityMinimumQuantityResponse = {
+  id: string;
+  item_object: InventoryItemObjectModel;
+  created_date: string;
+  min_quantity: number;
+  item: number;
+};
+
+interface FacilityObjectModel {
+  id: string;
+  name: string;
+  local_body: number;
+  district: number;
+  state: number;
+  ward_object: WardModel;
+  local_body_object: { id: number } & LocalBodyModel;
+  district_object: DistrictModel;
+  state_object: StateModel;
+  facility_type: string;
+  read_cover_image_url: string;
+  features: number[];
+  patient_count: string;
+  bed_count: string;
+}
+
+export type IPatientTransferRequest = {
+  facility?: string;
+  date_of_birth?: string;
+};
+export type IPatientTransferResponse = {
+  date_of_birth: string;
+  patient: string;
+  facility_object: FacilityObjectModel;
+};
+
+export type IInvestigationResponse = PaginatedResponse<{
+  id: string;
+  group_object: {
+    external_id: string;
+    name: string;
+  };
+  investigation_object: {
+    external_id: string;
+    name: string;
+    unit: string;
+    ideal_value: string;
+    min_value: number;
+    max_value: number;
+    investigation_type: string;
+    choices: string;
+  };
+  session_object: {
+    session_external_id: string;
+    session_created_date: string;
+    created_by: number;
+  };
+  value: number;
+  notes: string;
+  investigation: number;
+  group: number;
+  consultation: number;
+  session: number;
+}>;
+
+type BedObjectModel = BaseResponse & {
+  bed_type: number;
+  location_object: {
+    id: string;
+    facility: {
+      id: string;
+      name: string;
+    };
+    created_date: string;
+    modified_date: string;
+    name: string;
+    description: string;
+    location_type: number;
+  };
```

Comment:
Map this to use LocationModel instead

```suggestion
  location_object: LocationModel;
```